lib/primitives/mtx.js

```diff
@@ -1413,6 +1413,27 @@ class MTX extends TX {
 /**
  * Coin Selector
  * @alias module:primitives.CoinSelector
```

Comment:
Nice job on the jsdocs properties -- do we actually need this alias?

http://hsd-dev.org/hsd/module-primitives.CoinSelector.html

I'm not sure why it needs to be listed as a primitive since the code is not in the `primitves/` directory...

http://hsd-dev.org/hsd/module-primitives.html

Reply:
Also we're not describing the `options` here but I see that is missing from many classes in hsd, so that should be a future project...